Chiazokam/Politico
# #163351807 Create the create party endpoint

#### What does this PR do?
This PR tests and creates the Create Party Endpont

#### Description of Task to be completed?
- Set up the folder structure
- Write tests for all functions
- Create the Create party endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructure-create-party-163351807`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/parties `
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
	"partyName": "People Congress",
      "partyAddress": "12 Lincoln Ride",
      "partyLogo":"www.logo.com/url.jpg"
}
`
#### What are the relevant pivotal tracker stories?
[#163351807](https://www.pivotaltracker.com/story/show/163351807)

### Images
![image](https://user-images.githubusercontent.com/26940294/51748383-8c310f80-20ac-11e9-99d3-6ae43cd12748.png)
